public/app/features/dashboard/components/GenAI/GenAIButton.test.tsx

```diff
@@ -208,7 +208,9 @@ describe('GenAIButton', () => {
       // The tooltip keeps interactive to be able to click the link
       await userEvent.hover(tooltip);
       expect(tooltip).toBeVisible();
-      expect(tooltip).toHaveTextContent('Something went wrong');
+      expect(tooltip).toHaveTextContent(
+        'Failed to generate content using OpenAI. Please try again or if the problem persist, contact your organization admin.'
```

Comment:
This should read:
'Failed to generate content using OpenAI. Please try again, or if the problem persists, contact your organization admin.'